plugin/plugin.go

```diff
@@ -140,6 +140,20 @@ func Register(r *Registration) {
 	register.r = append(register.r, r)
 }
 
+// Disable disables a list of plugins by ids
+func Disable(plugins []string) {
```

Comment:
Could you handle this in `Graph` instead of another method or is there a reason for this?

Reply:
Yeah, will move the logic into `Graph`.

Reply:
Done.